zboxcore/sdk/chunked_upload_blobber.go

```diff
@@ -148,6 +148,7 @@ func (sb *ChunkedUploadBlobber) processCommit(ctx context.Context, su *ChunkedUp
 	wm.AllocationID = su.allocationObj.ID
 	wm.Size = size
 	wm.BlobberID = sb.blobber.ID
+	wm.Operation = sb.commitChanges[len(sb.commitChanges) - 1].GetOperation()
```

Comment:
It would panic if commitChanges is empty, though I see it will be setup before this function is called, but we'd better check if it's empty explicitly and return error if not